src/main/java/eu/dissco/backend/controller/DigitalMediaController.java

```diff
@@ -190,8 +190,7 @@ public ResponseEntity<JsonApiListResponseWrapper> getMasJobRecordForMedia(
       @Parameter(description = JOB_STATUS_OAS) @RequestParam(required = false) JobState state,
       @Parameter(description = PAGE_NUM_OAS) @RequestParam(defaultValue = DEFAULT_PAGE_NUM) int pageNumber,
       @Parameter(description = PAGE_SIZE_OAS) @RequestParam(defaultValue = DEFAULT_PAGE_SIZE) int pageSize,
-      HttpServletRequest request
-  ) throws NotFoundException {
```

Comment:
No NotFoundExcepion here? What if you call it for a media that does not exists?

Reply:
the underlying db query is: 

```
SELECT * from mas_job_records
WHERE target_id == {target_id}
```

If this response is empty (it returns a list), it could either mean the media doesn't exist, or more often, there aren't any scheduled mas jobs for that media. We could add a check to see if the media exists, but I don't think that adding an extra db call is a good idea. 

Besides, because this returns a list, if the query is empty, we don't throw a NPE, which is what this PR is trying to avoid

Reply:
Ok clear